content/spin/kubernetes.md

```diff
@@ -72,6 +72,12 @@ Azure AKS provides a straightforward and officially [documented](https://learn.m
 - The WASM/WASI node pools can't be used for system node pool.
 - The *os-type* for WASM/WASI node pools must be Linux.
 - You can't use the Azure portal to create WASM/WASI node pools.
+- AKS uses an older version of Spin for the shim, so you will need to change `spin_manifest_version` to `spin_version` in `spin.toml` if you are using a template-generated project from the Spin CLI
+- The RuntimeClass `wasmtime-spin-v0-5-1` on Azure maps to spin v1.0.0, and the `wasmtime-spin-v1` RuntimeClass uses an older shim corresponding to v0.3.0 spin.
+
+#### Note for Rust
+
+In Cargo.toml, the `spin-sdk` dependency should be downgraded to `v1.0.0-rc.1` in order to match the lower version running on the AKS shim
```

Comment:
@Mossaka I have a feeling we've had some problems with the 1.4 SDK because of wit-bindgen changes, but can no longer find the issue... do you know if 1.4 works?  That said, @davidgamero I don't want you to have to wait for us to figure it out, so feel free to take @Mossaka's recommended change, and he and I can sort out the 1.4 stuff in a separate PR.

Reply:
I haven't been able to get v1.4 to run on either of the runtimeclasses but i can verify!

Reply:
Thanks @davidgamero.  I reckon let's leave this for now because the backward compatibility story is a bit nuanced even within the 1.x series (@Mossaka I'm happy to clarify what the compatibility guarantee covers but TLDR is "not this I'm afraid").

Reply:
yep the 1.4 spin-sdk doesn't run on the wasmtime-spin-v0-5-1, but I suspect v1.3 does- deploying that now